noxfile.py

```diff
@@ -100,14 +100,15 @@
 
 # 'docfx' is excluded since it only needs to run in 'docs-presubmit'
 nox.options.sessions = [
```

Comment:
This is a templated file. See: https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/python_library/noxfile.py.j2

Please update [owlbot.py](https://github.com/googleapis/python-bigquery-magics/blob/23d54cd6721e362974f0119534fad726dc9a9608/owlbot.py#L63-L80) with this customization and/or update the template linked above.